nemo/collections/nlp/modules/common/tokenizer_utils.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import List, Optional
+
+import nemo
+from nemo.collections.common.tokenizers.char_tokenizer import CharTokenizer
+from nemo.collections.common.tokenizers.huggingface.auto_tokenizer import AutoTokenizer
+from nemo.collections.common.tokenizers.word_tokenizer import WordTokenizer
+from nemo.collections.nlp.modules.common.huggingface.huggingface_utils import (
+    get_all_huggingface_pretrained_lm_models_list,
+)
+from nemo.collections.nlp.modules.common.lm_utils import get_pretrained_lm_models_list
+from nemo.collections.nlp.modules.common.megatron.megatron_utils import get_megatron_tokenizer
+
+__all__ = ['get_tokenizer', 'get_tokenizer_list']
+
+
+def get_tokenizer_list() -> List[str]:
+    """
+    Returns all all supported tokenizer names
+    """
+    s = set(get_pretrained_lm_models_list())
+    s.update(set(get_all_huggingface_pretrained_lm_models_list()))
+    return ["sentencepiece", "char", "word"] + list(s)
+
+
+def get_tokenizer(
```

Comment:
Hmm, should we move this to common collection (together with HF utils)?
And to make this not dependent on NLP collection we can put lines like these:
```
from nemo.collections.nlp.modules.common.lm_utils import get_pretrained_lm_models_list
from nemo.collections.nlp.modules.common.megatron.megatron_utils import get_megatron_tokenizer
```
into try-except-finally block so that it doesn't throw exception if used from ASR without NLP installed?